internal/chain/transaction_test.go

```diff
@@ -38,7 +38,7 @@ func TestTxBuilder(t *testing.T) {
 	bgCtx := context.Background()
 	toAddress := common.HexToAddress("0xAb5801a7D398351b8bE11C439e05C5B3259aeC9B")
 	value := big.NewInt(1000)
-	txHash, err := txBuilder.Transfer(bgCtx, toAddress.Hex(), value)
+	txHash, err := txBuilder.TransferETH(bgCtx, toAddress.Hex(), value)
```

Comment:
Would it be better to add a test transaction for `TransferLSK`?

Reply:
I've already created [issue](https://github.com/LiskHQ/lsk-faucet/issues/4) to add unit test cases.